theme6/source/stylesheets/site.css.scss

```diff
@@ -1,159 +1,233 @@
+}
+
+.logo {
+  align-content: center;
+  display: flex;
+  justify-content: center;
+  width: 100px;
+}
+
+.social-media {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.sm {
+  border-radius: 3px;
+  border: 2px solid #72808c;
+  height: 35px;
+  width: 93px;
+}
+
+.fb {
+  background-position: center center;
+  background-repeat: no-repeat;
+  background: url("/images/fb.svg");
+}
+
+.twt {
+  background-position: center center;
+  background-repeat: no-repeat;
+  background: url("/images/twt.svg");
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.g {
+  background-position: center center;
+  background-repeat: no-repeat;
+  background: url("/images/g+.svg");
+}
+
+.newsletter {
+  align-items: flex-end;
+  color: #72808c;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  text-align: center;
+  text-align: right;
+}
+
+.newsletter .box-newsletter {
+  justify-content: center;
+}
+
+.newsletter input {
+  font-size: 15px;
+  height: 20px;
+  width: 300px;
+}
+
+.social-media-box {
+  align-content: center;
+  text-align: center;
+  width: 350px;
+}
+
+.social-media-icon-group button {
+  cursor: pointer;
+}
+
+.social-media-container {
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.information a {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  text-decoration: none;
+}
+
+.author {
+  color: #72808c;
+  display: block;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: center;
+}
+@media screen and (max-width: 1024px) {
+  .information a {
+    font-size: 14px;
+  }
+
+  .sm {
+    border-radius: 3px;
+    border: 2px solid #72808c;
+    height: 25px;
+    width: 60px;
+  }
+
+  .newsletter input {
+    font-size: 12px;
+    height: 15px;
+    width: 200px;
+  }
+
+  .newsletter p {
+    font-size: 15px;
+  }
+}
+@media screen and (max-width: 768px) {
+  .information {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .social-media-container {
+    display: block;
+    width: 100%;
+  }
+
+  .social-media-box {
+    display: block;
+    height: auto;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .social-media p {
+    font-size: 14px;
+    text-align: center;
+    width: 100%;
+  }
+
+  .social-media-icon-group {
+    display: inline-flex;
+    justify-content: center;
+  }
+
+  .twt {
+    margin-left: 3px;
+    margin-right: 3px;
+  }
+
+  .newsletter {
+    display: block;
+    display: block;
+    justify-content: center;
+    padding-bottom: 10px;
+    text-align: center;
+    transform: translateY(5px);
+    width: 100%;
+  }
+
+  .newsletter p {
+    font-size: 15px;
+    text-align: center;
+  }
+
+  .newsletter input {
+    background-color: #fbfbfb;
+    border-radius: 2px;
```

Comment:
Properties should be ordered background-color, border, border-radius, height, width